### PR TITLE
Disaster Recovery for Cluster MD (bsc#1168482)

### PR DESCRIPTION
**Pull request by taroth21**
### Description

https://bugzilla.suse.com/show_bug.cgi?id=1168482
https://jira.suse.com/browse/DOCTEAM-119

### Backports

- [x] To maintenance/SLEHA15SP2
- [x] To maintenance/SLEHA15SP1
- [x] To maintenance/SLEHA15
- [x] To maintenance/SLEHA12SP5
- [x] To maintenance/SLEHA12SP4

### CODE REVIEW

**Comment by taroth21** [comment]
@zzhou1 : Could you please have a look and review? 

Draft PDF Output is visible here:
image

**Comment by zzhou1** [comment]
LGTM

File: xml/ha_cluster_md.xml [hunk 1]
```diff
@@ -306,4 +306,28 @@ ARRAY /dev/md0 UUID=1d70f103:49740ef1:af2afce5:fcf6a489</screen>
    </step>
   </procedure>
  </sect1>
+
+ <sect1 xml:id="sec-ha-cluster-md-convert-raid">
+  <title>Assembling Cluster MD as normal RAID at the disaster recovery site</title>
+  <para>
+   In case of a disaster recovery, you might face the situation that you do not have
```
Comment (lproven) [inline xml/ha_cluster_md.xml:313]:
```suggestion
   In the event of disaster recovery, you might face the situation that you do not have
```

File: xml/ha_cluster_md.xml [hunk 2]
```diff
@@ -306,4 +306,28 @@ ARRAY /dev/md0 UUID=1d70f103:49740ef1:af2afce5:fcf6a489</screen>
...
+   a &pace; cluster stack in the infrastructure on the disaster recovery site, but
+   applications still need to access the data on the existing Cluster MD disks,
+   or from the backups.
+  </para>
+  <para>
+   You can convert Cluster MD RAID to a normal RAID by using the <option>--assemble</option>
```
Comment (lproven) [inline xml/ha_cluster_md.xml:319]:
```suggestion
   You can convert a Cluster MD RAID to a normal RAID by using the <option>--assemble</option>
```

**Review by lproven — CHANGES_REQUESTED** [review]
Very small suggestions: moving an article and changing one word.

**Comment by taroth21** [comment]
Merged now to get it into the localization drop for SP3, backports to follow tomorrow.

**Pull request merged**